docs/PATH_NOTATION.md

```diff
@@ -65,14 +65,209 @@ set({}, 'nested.1', 'new value')
 // }
 ```
 
-## :construction: This page is still a work in progress, the following sections will be written soon
-
 ## Bracket notation (property access)
+
+The bracket notation is the same as JavaScript [bracket notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_Accessors). It gives access to properties with an invalid identifier or dots in it.
+
+```js
+set({}, 'prop.[nested.1]', 'new value')
```

Comment:
Is the leading dot in front of the bracket intentional ? It is allowed but uncommon.